src/status_im/ui/screens/wallet/send/views.cljs

```diff
@@ -282,9 +282,11 @@
        [react/view styles/send-transaction-form
         [wallet.components/cartouche {:disabled? true}
          (i18n/label :t/message)
-         [components/amount-input {:disabled?     true
-                                   :input-options {:multiline     true
-                                                   :default-value data}} nil]]]]
+         [components/amount-input
+          {:disabled?     true
+           :input-options {:multiline true}
+           :amount-text   data}
```

Comment:
This is ok for a hotfix, but this message deserves its own component.

Reply:
Fully agree